pkg/entrypoint/entrypointer.go

```diff
@@ -267,3 +320,22 @@ func (e Entrypointer) WriteExitCodeFile(stepPath, content string) {
 	exitCodeFile := filepath.Join(stepPath, "exitCode")
 	e.PostWriter.Write(exitCodeFile, content)
 }
+
+// CheckForBreakpointOnFailure if step up breakpoint on failure
+// waiting breakpointExitPostFile to be written
+func (e Entrypointer) CheckForBreakpointOnFailure() {
```

Comment:
all the functions used in `CheckForBreakpointOnFailure` have test cases already. Since `os.Exit(1)` is used inside, no specific tests were added for this function.

Reply:
There is no coverage show in the report though. Is that because tests covering this are in a different module?

Reply:
added test cases for `CheckForBreakpointOnFailure`, now there are test coverage~